test/lit/wat-kitchen-sink.wast

```diff
@@ -3763,7 +3801,354 @@
   drop
  )
 
- ;; CHECK:      (func $call (type $21) (param $0 i32) (param $1 i64) (result f32)
+ ;; CHECK:      (func $string-new (type $59) (param $0 i32) (param $1 i32) (result stringref)
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (string.new_utf8_try
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (string.new_wtf8
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (string.new_lossy_utf8
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (string.new_wtf16
+ ;; CHECK-NEXT:   (local.get $0)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $string-new (param i32 i32) (result stringref)
+  local.get 0
+  local.get 1
```

Comment:
Why do I see `local.get 1` in the input but not in the output?

Reply:
This was a bug! Fixed now.